utils/moveImage.js

```diff
@@ -1,7 +1,14 @@
 const TweenMax = require("/public/cdnjs/TweenMax.min.js");
+
```

Comment:
this is Not related to this file I Just wanted to leave the comment here.

Can we remove the `GallerySimulation.jsx` if we no longer need it